src/ssmDocument/commands/createDocumentFromTemplate.ts

```diff
@@ -86,14 +88,31 @@ export async function createSsmDocumentFromTemplate(): Promise<void> {
     }
 }
 
-async function openTextDocumentFromSelection(item: SsmDocumentTemplateQuickPickItem): Promise<vscode.TextDocument> {
-    const template: object = getDocumentTemplate(item.docType)
+function yamlToJson(yaml: string): string {
+    const jsonObject = YAML.parse(yaml)
```

Comment:
We added a couple of starter templates for the "Create new document" action. These templates are defined in YAML. We don't expect these template to be invalid because we define them.

When the user executes "Create new document" action, it prompts him to select one of the starter templates. After selecting the template, it prompt him to select a document format (YAML or JSON). If the user selects JSON we do a conversion of the YAML content to JSON.